GTBitsOfGood/fair-districts-ga
# Create search bar component

## Add search bar component

Issue Number(s): Closes #24 

What does this PR change and why?

Created `SearchBar` component and updated the relevant pages (Legislator, Newspaper) to display the search bar and search the displayed tables in real-time.

### Checklist


### Related PRs

- Will need to make additional changes to incorporate the search bar into the volunteers page after completion.

### How to Test

1. Open legislator / newspaper pages
2. Create new legislators or newspapers through the modal
3. Type text into search bar and check results match expected behavior
